platform.js

```diff
@@ -130,7 +130,7 @@ module.exports = {
       return module.exports.spawn("taskkill /f /im javaw.exe");
     }
     else {
-      return module.exports.spawn("pkill -f " + module.exports.getJavaExecutablePath());
+      return module.exports.spawn("pkill -f Rise.*jar");
```

Comment:
@fjvallarino do we need to add quotes around `Rise.*jar`? If we don't, I'm worried bash might expand it to any filename that matches the `Rise.*jar` globbing pattern